code/framework/utils/src/main/java/io/cattle/platform/util/exception/LoggableException.java

```diff
@@ -0,0 +1,7 @@
+package io.cattle.platform.util.exception;
+
+import org.slf4j.Logger;
+
+public interface LoggableException {
+    void log(Logger log);
```

Comment:
Should I just provide 2 methods - logLevel(INFO, DEBUG, etc), and logStackTrace (boolean) - instead in this interface?